### PR TITLE
Make image smaller so that user can access GetStarted button.

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/DRsxTWfP/764-ipad-landscape-intro-wizard-first-screen-cut-off-cant-continue

🚩This is the temporary, not ideal fix. As soon as Gosia will be back, I will reach out to her to get images designed specifically for iPad.

### CODE REVIEW

**Review by SzymonGorowski — APPROVED** [review]
Approved 😄

**Review by anna1901 — CHANGES_REQUESTED** [review]
I don't like that it is stretched like that. I think it would be better to change how this image is added to this view and add it as a background image instead

**Comment by jankrzempek** [comment]
> I don't like that it is stretched like that. I think it would be better to change how this image is added to this view and add it as a background image instead

What do you mean by background, could you please elaborate or give an example?

**Review by anna1901 — COMMENTED** [review]
Add image as a background image using `.background()`. I'm not sure how I can elaborate on this. We can have a call if I should provide you some more information.

File: AirCasting/Assets.xcassets/Bitmap.imageset/Contents.json [hunk 1]
```diff
@@ -13,6 +13,14 @@
       "filename" : "Bitmap@3x.png",
       "idiom" : "universal",
       "scale" : "3x"
+    },
+    {
+      "idiom" : "ipad",
```
Comment (anna1901) [inline AirCasting/Assets.xcassets/Bitmap.imageset/Contents.json:18]: This shouldn't be here, right?

File: AirCasting/Onboarding/GetStarted.swift [hunk 2]
```diff
@@ -11,18 +11,25 @@ struct GetStarted: View {
...
+            logoImage
             descriptionText
             if featureFlagsViewModel.enabledFeatures.contains(.searchAndFollow) {
                 continueToAirNearYouScreenButton
             } else {
                 startButton
             }
-            Spacer()
-        }
+        }.background(
```
Comment (anna1901) [inline AirCasting/Onboarding/GetStarted.swift:22]: new line?

**Pull request merged**